terraform/eks/database.tf

```diff
@@ -128,4 +128,37 @@ resource "aws_dynamodb_table" "apm_test_table" {
     type = "S"
```

Comment:
For our one-step script for EKS, Kubernete, and EC2 setup, we need to write the relevant script to set up the mongodb database. The relevant code is here: https://github.com/aws-observability/application-signals-demo/tree/main/scripts. Note that the scripts are written in bash, so we probably need to setup the database using some aws cli commands.

Reply:
By setup, do you mean populate with pet types?

Reply:
I means to set up the mongodb database inside EC2 or as a aws service.